examples/spring-boot/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+spring:
+    main:
+        banner-mode: 'OFF'
+    jackson:
+        serialization:
+            WRITE_DATES_AS_TIMESTAMPS: false
+    datasource:
```

Comment:
it is a good practice to not put default JDBC parameters into the app, to avoid a mix of two.